pootle/static/js/shared/models/project.js

```diff
@@ -46,6 +42,10 @@ export const Project = Backbone.Model.extend({
     return l(['', 'projects', this.get('code'), 'admin', 'permissions', ''].join('/'));
   },
 
+  getFSUrl() {
+    return l(['', 'admin', 'projects', this.get('code'), 'fs', ''].join('/'));
```

Comment:
atm its superuser only - im not sure about project admins - i can imagine some sites would/not want project admins to change this